README.md

```diff
@@ -40,6 +40,7 @@ settings in cdk.json. To disable these, simply remove the context entries or set
 The settings are:
 
 - `enableLDSAccess`: if true, gives the LINZ Data Service/Koordinates access to the storage bucket.
+- `enableOpenTopographyAccess`: if true, gives the OpenTopography access to the storage bucket.
```

Comment:
"the" is superfluous here. Also, both of these settings should specify that they give *read-only* access.

Reply:
Fixed now. Thanks